lib/node_modules/@stdlib/math/base/special/logaddexp/examples/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var incrspace = require( '@stdlib/math/utils/incrspace' );
+var logaddexp = require( '@stdlib/math/base/special/logaddexp' );
```

Comment:
Project convention has been to use the package path in the README, while using `./../lib` for the actual example file. So, here, use `./../lib` for the second `require` path.